scientific_library/tvb/simulator/plot/base_plotter.py

```diff
@@ -34,29 +34,29 @@
 """
 
 import os
-import numpy
-from tvb.basic.logger.builder import get_logger
-from tvb.simulator.plot.config import CONFIGURED
 
 import matplotlib
-matplotlib.use(CONFIGURED.MATPLOTLIB_BACKEND)
-
+import numpy
 from matplotlib import pyplot
-pyplot.rcParams["font.size"] = CONFIGURED.FONTSIZE
-
 from mpl_toolkits.axes_grid1 import make_axes_locatable
-from tvb.simulator.plot.utils import ensure_list, generate_region_labels
+from tvb.basic.logger.builder import get_logger
+from tvb.simulator.plot.config import FiguresConfig, Config
+from tvb.simulator.plot.utils import generate_region_labels, ensure_list
+
+matplotlib.use(FiguresConfig().MATPLOTLIB_BACKEND)
+
+pyplot.rcParams["font.size"] = FiguresConfig.FONTSIZE
 
 
 class BasePlotter(object):
 
-    def __init__(self, config=CONFIGURED):
-        self.config = config
+    def __init__(self, config=None):
```

Comment:
The previous code allowed more unity in the configurations. Currently, the fresh Config() in case of missing init param is always "fresh"